### PR TITLE
feat: launch phoenix with evaluations

### PR DESCRIPTION
**Pull request by axiomofjoy**
adds support for launching phoenix with evaluations

### CODE REVIEW

**Review by RogerHYang — APPROVED** [review]

File: src/phoenix/session/evaluation.py [hunk 1]
```diff
@@ -33,24 +34,29 @@
...
 
 
-def add_evaluations(
-    exporter: HttpExporter,
-    evaluations: pd.DataFrame,
-    evaluation_name: str,
-) -> None:
-    index_names = evaluations.index.names
-    for index, row in evaluations.iterrows():
+def encode_evaluations(evaluations: Evaluations) -> Generator[pb.Evaluation, None, None]:
```
[the remark this answers is not in the document]
Comment (axiomofjoy) [reply src/phoenix/session/evaluation.py:37]: This is more readable, but less specific as a return type, I believe?

[the remark this answers is not in the document]
Comment (RogerHYang) [reply src/phoenix/session/evaluation.py:37]: i think they're equivalent in this context

[the remark this answers is not in the document]
Comment (RogerHYang) [reply src/phoenix/session/evaluation.py:37]: the other two None has to to with the receiver syntax of `yield` which we're not using, e.g.
```python
y = yield x
```

[the remark this answers is not in the document]
Comment (axiomofjoy) [reply src/phoenix/session/evaluation.py]: addressed in [3fd474d](https://github.com/Arize-ai/phoenix/pull/2095/commits/3fd474d4ed56fc2f47263475f1da7c2a68ab1af2)

**Pull request merged**